pkg/render/intrusion_detection_test.go

```diff
@@ -154,7 +154,7 @@ var _ = Describe("Intrusion Detection rendering tests", func() {
 		// Should mount ManagerTLSSecret for non-managed clusters
 		idc := rtest.GetResource(resources, "intrusion-detection-controller", render.IntrusionDetectionNamespace, "apps", "v1", "Deployment").(*appsv1.Deployment)
 		idji := rtest.GetResource(resources, "intrusion-detection-es-job-installer", render.IntrusionDetectionNamespace, "batch", "v1", "Job").(*batchv1.Job)
-		Expect(idc.Spec.Template.Spec.Containers).To(HaveLen(1))
+		Expect(idc.Spec.Template.Spec.Containers).To(HaveLen(2))
```

Comment:
I think we need a test for management clusters that asserts the webhook controller isn't installed.

From what I can tell, this test file is missing ManagementCluster test cases altogether.

Reply:
I was thinking about that too. I will create a new test case for management clusters. Although I'm not sure if it should belong here or in the `operator-cloud` codebase. I'll let you know when this is completed.

Reply:
It belongs here (operator-cloud is on freeze for new code)

Reply:
@caseydavenport Tests are now implemented. Pleas have another look.